src/style.css

```
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  margin-top: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+p {
+  margin: 0;
+}
```

```suggestion
h1,
p {
  margin: 0;
}
```
Don't repeat yourself. You may write like I've shown above and remove outer spacings from `h1` and `p` both